mne_bids_pipeline/_config.py

```diff
@@ -527,10 +554,30 @@
     ```
 """
 
-###############################################################################
-# MAXWELL FILTER PARAMETERS
-# -------------------------
-# done in 01-import_and_maxfilter.py
+# %%
+# ## Bad channel detection
+#
+# ---
+# tags:
+#   - preprocessing
+#   - raw
+#   - bad-channels
+# ---
+#
+# !!! warning
+#     This functionality will soon be removed from the pipeline, and
+#     will be integrated into MNE-BIDS.
+#
+# "Bad", i.e. flat and overly noisy channels, can be automatically detected
```

Comment:
Blocks of text like this were taken from the `.md` files and moved to the `_config.py`, which I think is a quality-of-life improvement for end users. We had little bits of this stuff in some places and some outdated info, and now it's documented nicely in the code and in the website the same way.